hazelcast-jet-sql/src/main/java/com/hazelcast/jet/sql/impl/schema/MappingCatalog.java

```diff
@@ -133,4 +141,24 @@ private Table toTable(Mapping mapping) {
                 mapping.fields()
         );
     }
+
+    @Override
+    public void registerListener(TableListener listener) {
+        listeners.add(listener);
+        storage.registerListener(new MappingStorage.EntryListenerAdapter() {
```

Comment:
Here we register a listener to the storage for each listener registered to mapping catalog. There should be only one listener registered to the storage.